sdcm/cluster.py

```diff
@@ -5497,6 +5497,8 @@ def node_startup(self, node, **kwargs):  # pylint: disable=unused-argument
         pass
 
     def install_scylla_manager(self, node):
+        if node.is_docker():
```

Comment:
If you really want to fail fast 

This should be checked in sct_config.py

Reply:
right, that would be faster)